modules/virt-about-nmstate.adoc

```diff
@@ -27,5 +27,5 @@ Node networking is monitored and updated by the following objects:
 
 [NOTE]
 ====
-If your {product-title} cluster uses OVN-Kubernetes as the network plugin, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
+If your {product-title} cluster uses OVN-Kubernetes as the network plugin, configuration changes to the interface is not allowed. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
```

Comment:
is>are since it refers to the plural changes earlier in the sentence:
```suggestion
If your {product-title} cluster uses OVN-Kubernetes as the network plugin, configuration changes to the interface are not allowed. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
```